node_modules/oae-core/comments/css/comments.css

```diff
@@ -40,6 +40,15 @@
 
 /* iPhone resolution */
 @media (max-width: 480px) {
+    .comments-widget #comments-container > .media > .media-body {
+        overflow: visible;
+    }
+
+    .comments-widget .comments-level-0 .comments-reply-container,
```

Comment:
@nicolaasmatthijs:

> Not that this is working as expected on mobile devices, but not on larger devices.

Was the first word intended to be "Note" instead of "Not"? I ask because the only changes in this PR are in the mobile device media query, so if it doesn't work on larger devices, that's been a problem all along.